### PR TITLE
Background image and background items

### PR DESCRIPTION
**Pull request by T-Damer**
- Added bg image for desktop and mobiles. Media query handles lazy loading automatically
- Added items for bg with minimal configuration, styled using tailwind

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -12,16 +12,18 @@ html {
...
 body {
   margin: 0;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
   font-family: 'JetBrains Mono', monospace;
   color: #efecd6;
-  scroll-behavior: smooth;
   min-height: 100vh;
   width: 100vw;
+  background-image: url('/img/blurry-shadows-lg.webp');
```
Comment (T-Damer) [inline src/index.css:25]: We can't add styles into `body` with tailwind

File: src/index.css [hunk 2]
```diff
@@ -33,6 +35,9 @@ body {
       )
       0 0/2rem 2rem;
   }
+  body {
+    background-image: url('/img/blurry-shadows-sm.webp');
```
Comment (T-Damer) [inline src/index.css:39]: LazyLoads if user uses mobile version

File: src/icons/IronSphere.tsx [hunk 3]
```diff
@@ -0,0 +1,419 @@
...
+        <g filter="url(#filter1_f_94_2823)">
+          <path
+            d="M38.3192 12.4931C44.9689 19.0186 41.7057 33.0087 39.2428 39.188C31.8543 37.1612 18.4626 32.544 24.004 30.2897C29.5454 28.0355 29.0836 11.455 28.16 3.44653C28.7757 3.74314 31.6696 5.96772 38.3192 12.4931Z"
+            fill="url(#paint2_linear_94_2823)"
+          />
+        </g>
+        <g filter="url(#filter2_f_94_2823)">
+          <path
+            d="M11.2344 5.24487C10.8164 5.89788 10.4153 6.50716 10.0303 7.07805C8.05897 10.9555 6.95122 17.7406 5.78776 21.5132C4.40648 25.992 4.40632 30.2352 5.78768 34.0069C6.32435 35.4722 9.32802 44.929 7.16904 42.9646C3.1419 39.3005 -0.609894 30.8261 1.23189 22.1041C2.6313 15.477 4.90281 14.682 10.0303 7.07805C10.3994 6.35213 10.7987 5.72812 11.2344 5.24487Z"
+            fill="#5390B8"
```
Comment (smyzin) [inline src/icons/IronSphere.tsx:45]: It's better to replace them with tailwind classes

**Review by smyzin — APPROVED** [review]
lgtm

**Pull request merged**